### ISSUE TITLE
[BUG]: Update Failure following Recent Commits

### ISSUE THREAD

**Issue by sinisterseed** [PCSX2/pcsx2#8276]
### Describe the Bug

Starting with version v1.7.4150, the update process appers to be failing consistently. The process advances almost all the way to the very end, but fails once it encounters a library which it cannot rename (a name which appears to be the same, anyway?)

I'm using the emulator's auto-updater to update, like I've always done until now. I could update just fine much earlier today, to version 1.7.4147, but that doesn't appear to be possible anymore following the recent commits, I can't go past the aforementioned version.

A screenshot of the error itself:

![gdgd](https://user-images.githubusercontent.com/57503728/221432147-a350ed26-4c88-4300-bd34-2fcbe89a4bf7.png)

Then, the process aborts and suggest downloading a fresh build from Github instead, citing a possible corruption as the cause.

### Reproduction Steps

Try updating from an older version using the emulator's auto-updater (a Qt x64 AVX2 build, specifically).

### Expected Behavior

Update proceeds as normal.

### PCSX2 Revision

v1.7.4147

### Operating System

Windows 10 (64bit)

**Comment by MrCK1** [comment]
Seems to be working as normal for me. Can you try clearing out your directories and starting with a fresh folder?

**Comment by refractionpcsx2** [comment]
The updater hasn't changed, nor anything related, please use the forum/discord for support issues. Thanks.

**Comment by sinisterseed** [comment]
> Seems to be working as normal for me (updated to 4150 from an older build). Can you try clearing out your directories and starting with a fresh folder?

Actually, it seems I managed to fix it, by dumb luck no less.

It crossed my mind to delete the offending file, and once I did I got an error about it still being in use ("In use? How? The update failed and the process isn't running anymore")

Well, actually, it turns out that it somehow was. Presumably, either my previous gaming session caused the emulator not to close successfully, or the update failure kept it running somehow. I found the executable still active under Background Processes in Task Manager, eating over half of my CPU and spiking the power usage metrics.

Quite curious, but for now I guess it's fixed. Thanks for replying.
